src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource.java

```diff
@@ -899,6 +870,26 @@ public GHPermissionType fetch(String username) throws IOException, InterruptedEx
                             boolean fork = !ghRepository.getOwner().equals(pr.getHead().getUser());
                             listener.getLogger().format("%n    Checking pull request %s%n",
                                     HyperlinkNote.encodeTo(pr.getHtmlUrl().toString(), "#" + number));
+                            Pattern pullRequestLabelRegexPattern = request.getPullRequestLabelRegexPattern();
```

Comment:
Nope nope nope. This should be implemented as a new trait in an extension plugin. I do not believe you need any changes to the core plugin to implement a trait with the feature you are looking for (but I am happy to merge changes that enable traits)